ahnlich/types/src/query.rs

```diff
@@ -69,6 +76,25 @@ pub enum Query {
     ListClients,
     Ping,
 }
+fn serialize_non_zero_usize<S>(value: &NonZeroUsize, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
They don't have serialise for usize? Not u64?

Reply:
@deven96  No,serde doesn't impl serialize for usize which makes sense because usize is based on system arch(32 or 64)

Reply:
That makes sense... resolved